osu.Game/Online/Leaderboards/Leaderboard.cs

```diff
@@ -170,36 +170,36 @@ protected Leaderboard()
         {
             InternalChildren = new Drawable[]
             {
-                new GridContainer
+                new OsuContextMenuContainer
```

Comment:
This should be inside the scroll container instead, otherwise the menu appears outside the masking area after a scroll:

![image](https://user-images.githubusercontent.com/1329837/87271599-a6fc7b80-c50e-11ea-8649-4271d05f3105.png)

Reply:
Need the user top score container to be inside too, so just adding `Masking = true` to `OsuContextMenuContainer` works.

Reply:
Sounds fine ^^